skins/laika/tests/unit/components/shared/ReceiptOptOut.spec.ts

```diff
@@ -7,7 +7,7 @@ localVue.use( Buefy );
 
 describe( 'ReceiptOptOut', () => {
 
-	it( 'is unchecked on initial render', () => {
+	it( 'is checked on initial render', () => {
```

Comment:
Now this test, together with the component name of `ReceiptOptOut` makes it look like it's opted out by default. Maybe it's better to call the component `ReceiptOptIn` now?